txpipe/diagnostics.py

```diff
@@ -25,12 +27,14 @@ class TXDiagnosticPlots(PipelineStage):
         ('g_T', PNGFile),
         ('snr_hist', PNGFile),
         ('mag_hist', PNGFile),
+        ('response_hist', PNGFile),
 
     ]
 
     config_options = {
         'chunk_rows': 100000,
-        'delta_gamma': 0.02
+        'delta_gamma': 0.02,
+        'psf_prefix': 'mcal_psf_',
```

Comment:
ok now I see that there are different PSF column names... this will probably be fixed once we merge the shear calibrator branch from Emily